Cargo.lock

```diff
@@ -7041,10 +7041,16 @@ dependencies = [
  "time",
 ]
 
+[[package]]
+name = "weezl"
+version = "0.1.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3e2bb9fc8309084dd7cd651336673844c1d47f8ef6d2091ec160b27f5c4aa277"
+
 [[package]]
 name = "wgpu-core"
 version = "0.6.0"
-source = "git+https://github.com/gfx-rs/wgpu#e72724a6e393503c73f37e86aa9317a5c62e32b8"
+source = "git+https://github.com/gfx-rs/wgpu?rev=e72724a6e393503c73f37e86aa9317a5c62e32b8#e72724a6e393503c73f37e86aa9317a5c62e32b8"
```

Comment:
Is it possible to update these to wgpu-core 0.6.5/wgpu-types 0.6.1 instead or is there an issue with using the current published versions in Servo?

Reply:
There are compilation errors using more recent git revisions. I have not attempted to investigate them at all.